src/components/approval-type/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bcrs-shared-components/approval-type",
-  "version": "1.0.31",
+  "version": "1.0.32",
```

Comment:
Paul, in the shared components repo, we don't manually update the version. We let Lerna do the magic in which when we publish the package (after PR has been merged), Lerna will automatically update the version (Sev is the one that has access to do the publishing).

Reply:
Ok I'll change that back

Reply:
Thanks! Resolving.

Reply:
The following people have admin permissions in bcrs-shared-components and can run the Lerna commands:
Argus
SRE (shared login)
Kial
Meng
Omid
Patrick
Richard
Travis
Severin
Thor
Vysakh